app/src/main/java/com/anytypeio/anytype/ui/templates/TemplateSelectFragment.kt

```diff
@@ -63,6 +75,13 @@ class TemplateSelectFragment :
     override fun onStart() {
         jobs += lifecycleScope.subscribe(vm.viewState) { render(it) }
         jobs += lifecycleScope.subscribe(vm.isDismissed) { if (it) exit() }
+        proceed(vm.toasts) { msg ->
+            Toast.makeText(
```

Comment:
There is an extension function `toast(...)`

Reply:
> toast(...)
I have a problem related to this method; I will try to investigate it later.